golem-worker-service-base/src/gateway_api_definition_transformer/mod.rs

```diff
@@ -0,0 +1,36 @@
+use crate::gateway_api_definition::http::HttpApiDefinition;
+pub use api_definition_transformer::*;
+pub use auth_transformer::*;
+pub use cors_transformer::*;
+
+mod api_definition_transformer;
+mod auth_transformer;
+mod cors_transformer;
+
+pub fn auth_transformer() -> Box<dyn ApiDefinitionTransformer> {
```

Comment:
Let's maybe get rid of the trait and just use plain functions here? Looks like this  is not extensible anyway